tox.ini

```diff
@@ -0,0 +1,36 @@
+[tox]
+envlist = 
+#     py36-django{11,20},
+#     flake8,
+    apidocs
+
+skip_missing_interpreters = True
+usedevelop = True
+
+# [testenv]
+# deps =
```

Comment:
We will need testing with previous version of django as well.

Reply:
As soon as we have 1 automated test run it can easily be expanded to the entire matrix, I've done the same for many of my projects. For example: https://github.com/WoLpH/django-tags-input/blob/master/tox.ini#L12-L21